community/server/src/test/java/org/neo4j/server/rest/RetrieveNodeIT.java

```diff
@@ -173,7 +173,7 @@ public void shouldGet404WhenRetrievingNonExistentNode() throws Exception
     {
         gen.get()
                 .expectedStatus( 404 )
-                .get( nodeUri + "00000" );
+                .get( nodeUri + "600000" );
```

Comment:
This looks like it came out of nowhere?

Reply:
I don't know why this hit us now but this test started to be flaky. If this test happens to run when there are no nodes int the database the `nodeUri` will be `http://localhost:51280/db/data/node/0` so adding "0000" will not do much good it will still point to an existing node with id 0 and we will not get a 404. The 60000 is of course completely random but it is a large number and it is unlikely we will have that many nodes in the database in this test